components/locale-provider/ku_IQ.tsx

```diff
@@ -0,0 +1,44 @@
+import Pagination from 'rc-pagination/lib/locale/en_GB';
```

Comment:
rc-pagination@1.15.0 added locale/ku_IQ

Reply:
May I request rc-pagination@1.15.0 in package.json? I am not sure about change logs from v1.14.0 to v1.15.0, there is no changelogs [here](https://github.com/react-component/pagination/releases). If there is no breaking changes then I will send a PR.

Reply:
Yes, No breaking changes and we will update rc-pagination to 1.15.0 in 3.1.4 in few day.